OpenRA.Mods.Common/Traits/ProductionParadrop.cs

```diff
@@ -0,0 +1,170 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Drawing;
+using System.Linq;
+using OpenRA.Activities;
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Deliver the unit in production via paradrop.")]
+	public class ProductionParadropInfo : ProductionInfo, Requires<ExitInfo>
+	{
+		[Desc("Cargo aircraft used. Must have Aircraft trait.")]
```

Comment:
You no longer need to have the second sentence (but can keep it since this feature isn't documented).
Instead you can give `ActorReference` a type that is required.
`[ActorReference(typeof(AircraftInfo))]`